pennylane/circuit_graph.py

```diff
@@ -117,15 +117,15 @@ def __init__(self, ops, obs, wires, par_info=None, trainable_params=None):
             op.queue_idx = k  # store the queue index in the Operator
 
             if hasattr(op, "return_type"):
-                if op.return_type is qml.measurements.State:
+                if op.return_type is State:
                     # State measurements contain no wires by default, but wires are
                     # required for the circuit drawer, so we recreate the state
                     # measurement with all wires
-                    op = qml.measurements.MeasurementProcess(qml.measurements.State, wires=wires)
+                    op = MeasurementProcess(State, wires=wires)
```

Comment:
There are two different syntax here, `MeasurementProcess(State, wires=wires)` and `qml.state(wires=wires)`. The same is true for the sample in the elif statement below. I think we should pick one and be consistent?

Reply:
I would definitely prefer `qml.state(wires=wires)`, but `qml.state` doesn't accept any arguments...

Reply:
Ahh ok the comment makes sense now